feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ConversationAutoCompletionE2eTest.kt

```diff
@@ -0,0 +1,70 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.conversation
+
+import de.tum.informatics.www1.artemis.native_app.core.common.test.EndToEndTest
+import de.tum.informatics.www1.artemis.native_app.core.common.test.DefaultTestTimeoutMillis
+import de.tum.informatics.www1.artemis.native_app.core.test.test_setup.DefaultTimeoutMillis
+import de.tum.informatics.www1.artemis.native_app.core.common.test.testServerUrl
+import de.tum.informatics.www1.artemis.native_app.core.model.account.User
+import de.tum.informatics.www1.artemis.native_app.feature.login.service.network.RegisterService
+import de.tum.informatics.www1.artemis.native_app.feature.metistest.ConversationBaseTest
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.junit.runner.RunWith
+import org.koin.test.get
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.util.Logger
+import java.util.Locale
+import kotlin.test.assertTrue
+import kotlin.test.assertEquals
+import kotlin.time.Duration.Companion.milliseconds
+
+
+@Category(EndToEndTest::class)
+@RunWith(RobolectricTestRunner::class)
+class ConversationAutoCompletionE2eTest : ConversationBaseTest() {
+
+    private val registerService: RegisterService get() = get()
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `auto-completion suggests users`() {
```

Comment:
Please give your test a name like the other names. For example "test GIVEN users are registered in a course WHEN requesting auto complete users THEN the registered users are returned"